test/webhook-proxy.test.js

```diff
@@ -46,6 +50,19 @@ describe('webhook-proxy', () => {
     })
   })
 
+  test('emits events with a valid signature', (done) => {
+    // This test will be done when the webhook is emitted
+    webhook.on('push', () => done())
+
+    const body = {action: 'foo'}
+
+    emit({
+      'x-github-event': 'push',
+      'x-hub-signature': webhook.sign('test'),
+      body
+    })
+})
+
```

Comment:
this is the test that is failing and I don’t quite understand what we are testing here? My understanding is that the `emit` function causes a request to the server that you started for testing purposes. Probot is not used here, the webhook API instance is not hooked up to receive any events. Can we simply remove this test?

Reply:
@gr2m thanks for the review - I'm cool to remove this test, I had the same thought but didn't want to remove it prematurely in case I was missing something.